common/components/add-another/template.njk

```diff
@@ -0,0 +1,49 @@
+{# Load in GOV.UK Components that are used within repeating blocks #}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+{% set itemName = params.itemName or t("item") %}
+{% set items = params.items %}
+
+{% for item in items %}
+  {% set outerLoop = loop %}
+
+  {% call govukFieldset({
+    classes: 'app-!-position-relative',
+    legend: {
+      text: itemName + " " + loop.index,
+      classes: 'govuk-fieldset__legend--m',
+      isPageHeading: false
+    }
+  }) %}
+    <div>
+      {{ govukButton({
+        html: t("actions::remove_item", {
+          name: itemName | lower,
+          index: loop.index
+        }),
+        classes: "govuk-button--secondary app-!-position-top-right",
+        name: "multiple-action",
+        value: "remove::" + params.name + "::" + loop.index0
```

Comment:
What happens if someone else removes an item and causes the index to get out of whack?

Would it be possible to use something to identify the actual blob of info to be zapped?

Reply:
Not sure what you mean? This loops over the items that exist in this session (that work is coming in #736)

So essentially this count is only ever for that user in that session.

Reply:
👍